uci/ComputePotentialEnergy.py

```diff
@@ -6,10 +6,10 @@
 import os
 import math
 
-class ComputePotentialEnergy():
+k = 1./(4.*numpy.pi*8.854187817620389e-12)
+impactFact = 1.0e-9**2 
```

Comment:
Why did you decide to make these global scope rather than class scope? Doesn't that lead to potential symbol collisions if the user of the library also defines a symbol `k`?